src/bin/pgcopydb/ld_test_decoding.c

```diff
@@ -171,15 +171,16 @@ parseTestDecodingMessageActionAndXid(LogicalStreamContext *context)
 			return false;
 		}
 
-		if (strcmp(header.table.nspname, "pgcopydb") == 0)
+		metadata->action = header.action;
+
+		/*
+		 * Check if the message should be filtered out based on namespace and relation name
+		 */
+		if (ShouldFilterOutMessage(&(privateContext->filters), header.table.nspname,
+								   header.table.relname))
 		{
-			log_debug("Filtering out message for schema \"%s\": %s",
-					  header.table.nspname,
-					  context->buffer);
```

Comment:
Maybe we should keep a `log_trace` about that decision?

Reply:
[62ecc5a](https://github.com/dimitri/pgcopydb/pull/614/commits/62ecc5aaf46db0797e3768e17c4f180bb20668e2)